exercises/practice/zebra-puzzle/.docs/instructions.md

```diff
@@ -12,20 +12,20 @@ The following 15 statements are all known to be true:
 1. There are five houses.
 2. The Englishman lives in the red house.
 3. The Spaniard owns the dog.
-4. Coffee is drunk in the green house.
+4. The person in the green house drinks coffee.
 5. The Ukrainian drinks tea.
 6. The green house is immediately to the right of the ivory house.
-7. The Old Gold smoker owns snails.
-8. Kools are smoked in the yellow house.
-9. Milk is drunk in the middle house.
+7. The snail owner likes to go dancing.
+8. The person in the yellow house is a painter.
+9. The person in the middle house drinks milk.
 10. The Norwegian lives in the first house.
-11. The man who smokes Chesterfields lives in the house next to the man with the fox.
-12. Kools are smoked in the house next to the house where the horse is kept.
-13. The Lucky Strike smoker drinks orange juice.
-14. The Japanese smokes Parliaments.
+11. The person who enjoys reading lives in the house next to the person with the fox.
+12. The painter's house is next to the house with the horse.
+13. The person who plays football drinks orange juice.
+14. The Japanese person plays chess.
 15. The Norwegian lives next to the blue house.
 
-Additionally, each of the five houses is painted a different color, and their inhabitants are of different national extractions, own different pets, drink different beverages and smoke different brands of cigarettes.
+Additionally, each of the five houses is painted a different color, and their inhabitants are of different national extractions, own different pets, drink different beverages and engage in different hobbies.
```

Comment:
Probably also needs exercise update right?

Reply:
yesss

Reply:
I think the answer is still the same and the text has just been reworded, at least that's what I gather from looking at the `problem-spec`.

However, I do hate the fact that most solutions to this exercise are just the hardcoded answer that people looked up. Maybe I'll open a separate forum thread to discuss if anything can be done about it.